docs/index.md

```diff
@@ -0,0 +1,8 @@
+Try `jupyterlab-plugin-playground` in your browser right now [![try-badge]][try]
```

Comment:
hm... not clickable in rtd... also maybe throw a `{hint}` around it...

Reply:
good catch :+1:

Reply:
I tried a couple of things to get the link to work, not sure whether it's because the file is generated.